### ISSUE TITLE
Load and plot picks from file

### ISSUE THREAD

**Issue by DaloroAT** [DaloroAT/first_break_picking#24]
Feature request described here

https://github.com/DaloroAT/first_breaks_picking/issues/21#issuecomment-1602103271

**Comment by HHTC0** [comment]
Byte 237 worked fine. Thanks for that. Are you planning to release next update soon with these functionalities?

**Comment by DaloroAT** [comment]
Good!

New issues will be solved in next update. But I don't know when I will do this task.

Are you picking something and want to use it instead of existing professional solutions? Because right now it's just a demo to show the possibility of neural network + free of charge. It's not professional software.

**Comment by DaloroAT** [comment]
If you are going to use it professionally, I can make these fixes early

**Comment by HHTC0** [comment]
Yes, that would be really great. Yes, I have an upcoming project and I am planning to use this package for it. Thank you

**Comment by DaloroAT** [comment]
Hey @HHTC0 I published *beta* version with new functionality. 

Details are here: 
https://github.com/DaloroAT/first_breaks_picking/pull/29#issuecomment-1641457226

**Comment by HHTC0** [comment]
@DaloroAT just noticed that it won't allow editing/adjustment of loaded picks from headers. Can you allow that?

**Comment by DaloroAT** [comment]
@HHTC0 I can try. Is it urgent? Do you need it today?

Looks like you need some functionality today. If you need faster reply or want to discuss other functionality, you can write to my own e-mail (you can find it on my page profile) to share Whatsapp, Telegram, etc profiles.

**Comment by HHTC0** [comment]
@DaloroAT the loading is apparently correct. However, the main functionality I was after is ability to edit the loaded time picks. But I saw your comment that is not possible. 

I will drop you a message on Whatsapp. Thanks
